backend/core/interactem/core/nats/consumers.py

```diff
@@ -23,7 +23,7 @@
 logger = get_logger()
 
 AGENT_CONSUMER_CONFIG = ConsumerConfig(
-    deliver_policy=DeliverPolicy.LAST_PER_SUBJECT,
+    deliver_policy=DeliverPolicy.NEW,
```

Comment:
What does the policy change do?

Reply:
When reconnecting after a timeout, LAST_PER_SUBJECT would read in the last message and restart containers. With NEW we will only get new messages.